src/main/java/oracle/r2dbc/OracleR2dbcOptions.java

```diff
@@ -0,0 +1,224 @@
+/*
+  Copyright (c) 2020, 2021, Oracle and/or its affiliates.
+
+  This software is dual-licensed to you under the Universal Permissive License
+  (UPL) 1.0 as shown at https://oss.oracle.com/licenses/upl or Apache License
+  2.0 as shown at http://www.apache.org/licenses/LICENSE-2.0. You may choose
+  either license.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+package oracle.r2dbc;
+
+import io.r2dbc.spi.Option;
+import oracle.jdbc.OracleConnection;
+
+import java.util.concurrent.Executor;
+import java.util.concurrent.ForkJoinPool;
+
+/**
+ * Extended {@link Option}s supported by the Oracle R2DBC Driver.
+ */
+public final class OracleR2dbcOptions {
+
+  private OracleR2dbcOptions() {}
+
+  /**
+   * Extended {@code Option} that specifies an Oracle Net Connect Descriptor
+   * of the form "(DESCRIPTION=...)"
```

Comment:
This can also be an alias, right?

Reply:
Also I believe the DescriptorURL code sample needs to be updated.

Reply:
The latest commit has fixed these issues.